src/app/beer_garden/scheduler.py

```diff
@@ -510,10 +521,25 @@ def get_job(job_id: str) -> Job:
     return db.query_unique(Job, id=job_id)
 
 
-def get_jobs(filter_params: Dict = None) -> List[Job]:
+def get_jobs(filter_params: Optional[Dict] = None) -> List[Job]:
     return db.query(Job, filter_params=filter_params)
 
 
+def get_some_jobs(filter_params: Optional[Dict] = None) -> JobDefinitionList:
```

Comment:
I think once `JobDefitionList` goes away you lose the need for this method.  `get_jobs` although mapped to the "JOB_READ_ALL" operation actually allows you to get "some" jobs by providing a filter.

Reply:
Changed.